src/frontend/src/views/Data/table-headers/commonSampleHeaders.tsx

```diff
@@ -117,7 +117,7 @@ export const QC_METRICS_HEADER: Header = {
       linkText: "Learn more.",
     },
     regularText:
-      "Overall QC score from Nextclade which considers genome completion and screens for potential contamination and sequencing or bioinformatics errors.",
+      "Overall QC score from Nextclade which considers genome completion and screens for potential contamination and sequencing or bioinformatics errors. ",
```

Comment:
I feel like this kind of forgetting-to-drop-a-terminating-space-char for the various strings is going to wind up being common. With the other change here, I think I would probably save the file and verify on my local browser, so I'd notice missing that weird terminating JSX space. But for creating/updating a set of strings, I think I'd be a lot more likely to not see the issue, just save my work, and make a PR. I wonder if it would be less accident-prone / clearer to have wherever the interpolation happens handle the extra space instead?

I'm **not** actually suggesting this change for this PR. It just popped into my head as a mistake we might regularly make as we expand to having more sets of strings for various pathogens, so something I wanted to call out. If we notice ourselves regularly bumping our knees on this coffee table, we should come back and see if we can tweak the interpolation to make the string look more like what you'd expect at a glance.

Reply:
Ya, I agree with this. It's the placement of the following link that actually causes weird spacing anyway.